<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodageChaine</class>
 <widget class="QWidget" name="DecodageChaine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DecodageChaine</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelTexte">
       <property name="text">
        <string>Texte à décoder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTexteADecoder"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDecoder">
       <property name="text">
        <string>Décoder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelNom">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNom"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelPrenom">
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPrenom"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelAge">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAge"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labelPoids">
       <property name="text">
        <string>Poids</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPoids"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="labelTaille">
       <property name="text">
        <string>Taille</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTaille"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>DecodageChaine</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>694</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
